stripe/src/main/java/com/stripe/android/view/CheckoutActivity.kt

```diff
@@ -1,15 +1,78 @@
 package com.stripe.android.view
 
 import android.os.Bundle
+import android.view.View
 import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.lifecycleScope
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.bottomsheet.BottomSheetBehavior.STATE_HIDDEN
 import com.stripe.android.databinding.ActivityCheckoutBinding
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
 
 internal class CheckoutActivity : AppCompatActivity() {
     private val viewBinding by lazy {
         ActivityCheckoutBinding.inflate(layoutInflater)
     }
+    private lateinit var bottomSheetBehavior: BottomSheetBehavior<View>
```

Comment:
slightly prefer `private val by lazy` but not a blocker

Reply:
oh yes. that is nicer. I was just copying some of the bottom sheet examples and never revisited this